src/helpers/is-element-in-viewport.ts

```diff
@@ -1,12 +1,14 @@
+interface IViewportSpacing {
+  top: number;
+  right: number;
+  bottom: number;
+  left: number;
+}
+
 /**
  * Calculates if the given DOM element is visible in the viewport.
- *
- * @param {Node} element - The DOM element which needs to be tested.
- * @param {Object} viewportSpacing - Allows to define additional inner spacings for the viewport.
- *
- * @returns {Boolean}
  */
-export default function isElementInViewport(element, viewportSpacing = { top: 10, right: 0, bottom: 10, left: 0 }) { // eslint-disable-line object-curly-newline
+export default function isElementInViewport(element: HTMLElement, viewportSpacing: IViewportSpacing = { top: 10, right: 0, bottom: 10, left: 0 }): boolean { // eslint-disable-line object-curly-newline, max-len
```

Comment:
Please move the fallback fro `viewportSpacing` to a variable, so we get a shorter line (remove max-len).